createlinks

```diff
@@ -96,3 +96,12 @@ templates2events("/etc/backup-data.d/freepbx.include",$event);
 event_actions('nethserver-sssd-save', qw(
     nethserver-freepbx-conf-users       60
 ));
+
+#--------------------------------------------------
+# actions for post-restore-data event
+#--------------------------------------------------
```

Comment:
Not really necessary, but it's better to respect the conventions used inside this file:
```suggestion
#
# post-restore-data event
#
```